scripts/Deploy.s.sol

```diff
@@ -0,0 +1,79 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.8;
+
+import "forge-std-1.9.6/src/Script.sol";
+
+import "../src/inputs/InputBox.sol";
+import "../src/portals/EtherPortal.sol";
+import "../src/portals/ERC20Portal.sol";
+import "../src/portals/ERC721Portal.sol";
+import "../src/portals/ERC1155SinglePortal.sol";
+import "../src/portals/ERC1155BatchPortal.sol";
+import "../src/consensus/authority/AuthorityFactory.sol";
+import "../src/consensus/quorum/QuorumFactory.sol";
+import "../src/dapp/ApplicationFactory.sol";
+import "../src/dapp/SelfHostedApplicationFactory.sol";
+import "../src/delegatecall/SafeERC20Transfer.sol";
```

Comment:
It's good practice to import only what you need.
Please run `forge lint` to make sure there are no warnings.

```solidity
import {IInputBox} from "src/inputs/IInputBox.sol";
import {InputBox} from "src/inputs/InputBox.sol";
```